recipes/pygccxml/recipe.yaml

```
@@ -0,0 +1,43 @@
+context:
+  version: "3.0.2"
+  python_min: "3.10"
+
+package:
+  name: pygccxml
+  version: ${{ version }}
+
+source:
+  - url: https://pypi.org/packages/source/p/pygccxml/pygccxml-${{ version }}.tar.gz
+    sha256: e1a8c738765f4eb2819cc7439975631f838e59a1cf3fa49896e0a1967c5e2bee
+
+build:
+  script: ${{ PYTHON }} -m pip install .
+  noarch: python
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - setuptools
+    - pip
+  run:
+    - python >=${{ python_min }}
```

```suggestion
    - python >=${{ python_min }}
    - castxml
```

`pygccxml` has a run dependency on castxml (undeclared in pypi metadata, as `castxml` is not available on PyPI), I would add it as soon as https://github.com/conda-forge/staged-recipes/pull/30227 is merged.